pkg/ccl/backupccl/alter_backup_schedule.go

```diff
@@ -115,6 +115,12 @@ func doAlterBackupSchedules(
 		return err
 	}
 
+	if s.fullJob == nil {
+		// This can happen if a user calls DROP SCHEDULE on the full schedule.
+		// TODO(benbardin): Can we prevent this state from occurring in the first place?
+		return errors.Newf("no full backup schedule found, drop this schedule and recreate")
```

Comment:
IIUC this case can only occur if a full is dropped, and the user alters its inc?

In that case maybe this would be a clearer call to action to the user:

"schedule %d does not have a corresponding full backup schedule; drop schedule %d and recreate"

Reply:
I like this, thanks!